en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,859 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title: Asgardeo - Action Management Rest API
+  description:
+    "This is the RESTful API for managing action configuration for an organization in WSO2 Asgardeo. \n
+    This API allows creating, retrieving, updating and deleting the action configurations. \n\n
+    To access Action Management APIs in Asgardeo, you need to first [get an access token](https://wso2.com/asgardeo/docs/apis/authentication/#get-an-access-token) from your organization."
+servers:
+  - url: 'https://api.asgardeo.io/t/{organization-name}/api/server/v1'
+security:
+  - OAuth2: []
+paths:
+  /actions/types:
+    get:
+      tags:
+      - Actions Endpoint
+      summary: List Action Types
+      description: "This API returns the detailed summary of the action types.\n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_view``\n\n"
+      operationId: getActionTypes
+      responses:
+        '200':
+          description: Action Types are found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionTypesResponse'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl --location 'https://api.asgardeo.io/t/{organization-name}/api/server/v1/actions/types' \
+            -H 'Authorization: Bearer {bearer_token}'
+
+  /actions/{actionType}:
+    post:
+      tags:
+        - Actions Endpoint
+      summary: Add Action
+      description: "This API creates an action and returns the action details along with the action's unique ID.\n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_create``\n\n"
+      operationId: createAction
+      parameters:
+        - name: actionType
+          in: path
+          description: Name of the Action Type.
+          required: true
+          schema:
+            enum:
+              - preIssueAccessToken
+              - preUpdatePassword
+              - preUpdateProfile
+              - preRegistration
```

Comment:
Better not have unsupported types as of now

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/ee4f5ca7c24d3d8dcaca4943b7b5aadb6e16e7fc